modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -47,64 +54,56 @@ You can use the information in this topic to publish Mule 3 and Mule 4 assets in
-  <mule.version>3.9.0</mule.version>
-  <mule.tools.version>1.2</mule.tools.version>
-  <type>app</type>
- </properties>
- ...
-----
+`+https://maven.eu1.anypoint.mulesoft.com/api/v3/organizations/BUSINESS_GROUP_ID/maven+`
 +
-*Notes:*
+. In your project's POM file, include the `mule-maven-plugin` and set the correct classifier (in this example `mule-application-example`).
 +
-** *Important:* The `<classifier>` element must be set to make the asset visible in Exchange.
-** The `name` property represents the visible name of the asset. If you are publishing the first version of a GA (GroupId and AssetId combination), the asset name is taken from the POM file. If there is already at least one version of the GA, the `name` property is ignored. You can change the name in the asset portal as described in xref:to-describe-an-asset.adoc[Describe an Asset].
-** The `description` property represents the description of the asset. If you are publishing the first version of a GA (GroupId and AssetId combination), the asset description is taken from the POM file. If there is already at least one version of the GA, the `description` property is ignored. You can change the description in the asset portal as described in xref:to-describe-an-asset.adoc[Describe an Asset].
-** Mule 3 applications can be deployed setting the property <type> to `app`. The asset won't be displayed in the Exchange asset list.
-** Mule 3 `templates` and `examples` should be deployed using Anypoint Studio 6.3 or above.
-** For Mule 3 or Mule 4 connectors, do not specify a `<type>` value. The Maven Facade determines the type automatically.
-** For Mule 4 assets, do not specify the <type> element. In the `mule-maven-plugin` configuration, set `<classifier>` as `mule-application` (default), `mule-application-template`, `mule-application-example` or `mule-policy`.
+In this example, it is not necessary to set the asset type because it is inferred from the classifier.
 +
 For example:
 +
 [source,xml,linenums]
 ----
 ...
-<plugin>
-    <groupId>org.mule.tools.maven</groupId>
-    <artifactId>mule-maven-plugin</artifactId>
-    <version>${mule.maven.plugin.version}</version>
-    <extensions>true</extensions>
-    <configuration>
-        <classifier>mule-application-example</classifier>
-    </configuration>
-</plugin>
+<name>Hello World Application Example</name>
+<description>A mule application example uploaded using Exchange Maven Facade v3</description>
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.mule.tools.maven</groupId>
+            <artifactId>mule-maven-plugin</artifactId>
+            <version>3.5.0</version>
+            <extensions>true</extensions>
+            <configuration>
+                <classifier>mule-application</classifier>
+            </configuration>
+        </plugin>
+    </plugins>
+</build>
 ...
 ----
 +
+*Notes:*
++
+** *Important:* The `<classifier>` element must be set to make the asset visible in Exchange.
```

Comment:
Avoid using bold for something important; use a notice format instead. However, I suggest making importance obvious by placement and context within the running narrative instead. In this case, I wonder if this is a prerequisite? Using that label, or calling it a "requirement," emphasizes the importance.

This could also be written in active voice.